.github/workflows/index-validation.yaml

```diff
@@ -0,0 +1,32 @@
+name: Validate index files
+
+on:
+  push:
+    branches: ["main"]
```

Comment:
What would be the difference there? Not familiar with the idea of dependencies between workflows.

Reply:
The doc pages shouldn't be published if this is broken, so we should require this to pass before that job starts. With this independent trigger, it will just run and fail, while the other job publishes the page on the broken state without caring.